core/FoundDefinitions.h

```diff
@@ -156,11 +156,11 @@ struct FoundMethod final {
     Flags flags;
     CheckSize(Flags, 1, 1);
     std::vector<core::ParsedArg> parsedArgs;
-    std::vector<uint32_t> argsHash;
+    uint32_t argsHash;
 
     std::string toString(const core::GlobalState &gs, const FoundDefinitions &foundDefs, uint32_t id) const;
 };
-CheckSize(FoundMethod, 80, 8);
+CheckSize(FoundMethod, 64, 8);
```

Comment:
This is a nice size reduction!

You should be able to get this down a little further if you swap `parsedArgs` and `argsHash`, so the latter is packed into the same 8 bytes that `flags` currently occupies`.

Reply:
I'm not able to get core::FoundMethod any smaller than 64 bytes, am i doing something wrong?

I think that it has 57 bytes worth of information and then aligns to 8 bytes, so it pads the whole thing up to 64 instead of 60

I'm going to land this for now to unblock future changes, and if we want to tweak this in a future change we should feel free to.

Luckily these structures are transient (only used by namer) so they don't contribute a particularly large memory pressure at the moment (unless i'm mistaken).